sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_operations/_patch.py

```diff
@@ -26,6 +26,31 @@
 
 
 class LogsIngestionClientOperationsMixin(GeneratedOps):
+
+    @overload # type: ignore[override]
```

Comment:
I'm honestly not even sure if there's much benefit to having the overloads. All they do is separate out the `IO` and `JSON`, which I think was already conveyed from the `Union` type in the base method signature.

Reply:
That was my thought as well, and initially I just removed the generated overloads without adding these new ones. Then I thought to try and be consistent with the DPG generated overloads which seemingly follow the same pattern. Not sure if there was reasoning behind this in the generation.

I do agree that no new typing information is conveyed with these overloads, and it would be cleaner without them. So, I'd be fine with removing them. WDYT?

Reply:
Removed!